src/amp/components/BodyArticle.tsx

```diff
@@ -108,6 +114,8 @@ export const Body: React.FC<{
 
             {elements}
 
+            {epic}
```

Comment:
because anything that evaluates to `false` in JSX is removed
```suggestion
            (data.pageId === epicTestPageId) && <Epic />
```

Reply:
@liywjl this caused unexpected problems, so I've reverted these changes in commit 750bfce. The conditional logic is only temporary, so I hope this is ok with everyone 🙂